scripts/parse_specs.py

```diff
@@ -529,15 +530,20 @@ def _generate_meta(d, ordinal, meta):
         if 'enum' == type:
             value = -1
             max_value = -1
+            bit_mask = 0
             meta[type][name]['etors'] = []
             for idx, etor in enumerate(d['etors']):
                 meta[type][name]['etors'].append(etor['name'])
                 value = _get_etor_value(etor.get('value'), value)
+                if type_traits.is_flags(name):
+                    bit_mask |= value
```

Comment:
This will generate a bit mask for all flags defined in the spec and will handle if there are gaps.